tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbPartitionStorageTests.cs

```diff
@@ -5,16 +5,26 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.IO;
+using System.Net;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Azure.Cosmos;
 using Microsoft.Bot.Builder.Adapters;
 using Microsoft.Bot.Builder.Dialogs;
 using Microsoft.Bot.Builder.Tests;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Newtonsoft.Json.Linq;
+using static Microsoft.Bot.Builder.Azure.CosmosDbPartitionedStorage;
```

Comment:
We should avoid "using static".

Reply:
This is `static` because of the `[assembly: InternalsVisibleTo("Microsoft.Bot.Builder.Azure.Tests")]`. There's a couple of "fixes" for this:

1. Make `DocumentStoreItem` public.
2. Remove that `using` statement entirely and replacing `DocumentStoreItem` with `Microsoft.Bot.Builder.Azure.DocumentStoreItem` in the 12 places that it occurs.
  - Note: I think because of the `InternalsVisibleTo`, it doesn't allow me to replace it with simply `using Microsoft.Bot.Builder.Azure`